### ISSUE TITLE
LESS, SASS, and Stylus compilers don't respect outdated and force params

### ISSUE THREAD

**Issue by ipartola** [cyberdelia/django-pipeline#446]
This greatly slows down the development server since a new subprocess is fired on every template render for every file that needs to be compiled. I believe the fix is just to add the `if not outdated and not force: return` line that's already in the other compilers and in the docs.

**Comment by cyberdelia** [comment]
This is because of sass ``@import``, which breaks this system.

**Comment by ipartola** [comment]
Is this documented somewhere? Sorry, just trying to reduce the 10 second template render times here, where DEBUG = True.

**Comment by cyberdelia** [comment]
Not really. You might be able to leverage sass compiler own cache directly.
